pages/linux/groupmod.md

```diff
@@ -0,0 +1,11 @@
+# groupmod
+
+> Modify existing user groups in the system.
+
+- Change the group name:
+
+`groupmod -n {{new-group-name}} {{old-group-name}}`
+
+- Change the group id:
+
+`groupmod -g {{group-id}} {{existing-group-name}}`
```

Comment:
Perhaps it will be clearer if you mention {{new_group_id}}, to match with the previous example. I had to check the man pages because I was not sure the group_id referred to the old group_id or the new group_id.